lib/board.rb

```diff
@@ -36,14 +36,22 @@ def winning_player_exists?(player1_mark, player2_mark)
   end
 
   def complete?
-    available_squares.zero?
+    available_squares_quantity.zero?
   end
 
   def clear_squares
     @squares = [1, 2, 3, 4, 5, 6, 7, 8, 9]
   end
 
-  def available_squares
+  def available_squares_quantity
```

Comment:
Maybe `total_free_squares`?

Reply:
`total_available_squares` is a better name for this method. Thanks, Matt